tests/runtime/basic

```diff
@@ -158,6 +158,22 @@ EXPECT hardware:cache-misses:
 EXPECT tracepoint:xdp:mem_connect
 TIMEOUT 5
 
+NAME it lists given search pattern
+RUN {{BPFTRACE}} -l hardware:cache-misses
+EXPECT hardware:cache-misses:
+TIMEOUT 5
+
+NAME it lists given search pattern
+RUN {{BPFTRACE}} -l runtime/scripts/hello_world.bt
+EXPECT special::
+TIMEOUT 5
+
+NAME errors on non existent file
+RUN {{BPFTRACE}} -l non_existent_file.bt
+EXPECT ERROR: failed to open file 'non_existent_file.bt': No such file or directory
```

Comment:
This has a different error because it's now treated as a search right (removed the ".bt" check) ?